lib/events/events.proto

```diff
@@ -99,6 +106,10 @@ message SessionStart {
 
     // TerminalSize is expressed as 'W:H'
     string TerminalSize = 6 [ (gogoproto.jsontag) = "size,omitempty" ];
+
+    // Kubernetes has information about a kubernetes cluster, if applicable.
```

Comment:
Kubernetes -> KubernetesClusterMetadata

Reply:
`Kubernetes` is the field name, i'll leave as is